provision/ansible/README.md

```diff
@@ -0,0 +1,16 @@
+## Ansible Dependencies
+
+In order to successfully run the Ansible scripts defined herein, you'll need to install the required package dependencies. Do this using the following commands:
+
+Install the pip dependencies. You'll need to have python-pip installed on your deployment machine. Though not required, it's recommended you run the following command inside a [Python virtualenv](https://virtualenvwrapper.readthedocs.io/en/latest/):
+
+```sh
+pip install -r requirements/ansible.pip
+```
+
+Install the [Ansible Galaxy](https://galaxy.ansible.com/docs/) dependencies:
+
+```sh
+mkdir -p vendor/roles
+ansible-galaxy install -p vendor/roles -r requirements/galaxy.yml
```

Comment:
@jasonrogena :  When I try to run this on my machine (OS X.11) I'm getting the following error:

```
ansible-galaxy install -p vendor/roles -r requirements/galaxy.yml
Traceback (most recent call last):
  File "/usr/local/bin/ansible-galaxy", line 67, in <module>
    import ansible.constants as C
  File "/Users/Josh-LT/Library/Python/2.7/lib/python/site-packages/ansible/constants.py", line 11, in <module>
    from jinja2 import Template
ImportError: No module named jinja2
```

I ran `sudo pip install jinja2`, which succeeds, however when I run the ansible command again I get the same error.  I'm not using virtualenv which is not the best idea, however I'm wondering if we're missing something.  Perhaps we can have another dev try this with their machine and see if it works?